### ISSUE TITLE
Very difficult to land on the “Mine” radio button .mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#150]
Screengrab/videos (insert file names here)
2020 08 25, Very Difficult to Land on the “Mine” Radio Button 
[2020 08 25, Very difficult to land on the “Mine” radio button .mp4.zip](https://github.com/CityOfBoston/Iterators/files/5125387/2020.08.25.Very.difficult.to.land.on.the.Mine.radio.button.mp4.zip)

Link
https://registry.boston.gov/birth

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MasOs 10.15.5 > Chrome 84

Trusted Tester Test
N/a

Brief Description
Trying to land on the “Mine” button in order to purchase birth certificate, very difficult.

Steps to reproduce
1)	00:00, Go to the site, using the like listed above.
2)	00:07, Start to tab through the site, starting at City of Boston link.
3)	00:12, Tabbing from link titled Birth Certificates to “Mine”. Note: Landing on the “Mine” radio button does not make it Active.  You cannot hit “Enter” to return; you must use Up/Down arrows to activate the radio button. Therefore, this is a bug. 
a)	Note: A suggestion would be to use the Return Key to activate the Radio Buttons.
4)	00:21, Tabbing from the “Mine” radio button advances to birth@boston.gov. 
b) 	Note: The Entire Radio Button Segment, both “Mine” and “Someone’s Else” is not active and therefore skipped when moving through the website. The site does not indicate what are the “Required fields”. This, is also a bug as the user has no idea what data is required.
5) 00:25, Use the Shift+tab feature tab backwards and land on the “Someone Else” button. This is still not an active button.
6) 00:45, Use the Up/Down arrows to activate the Radio button Module, which finally opens the menu and view the next question regarding whose birth certificate you are ordering. 

Desired Result
•	Able to use the keyboard and land on the radio button module and make them active

Actual Result
•	Unable to use the keyboard and land on the radio button module and make them active

**Comment by subaha-cob** [comment]
@andrebtlima This will require going into registry app. For now, I will push this back until I get into Apps. Sending back to Triage.

**Comment by david-iterators** [comment]
Hi @andrebtlima We found that we can use the space key in order to select the "Mine" radio button and the other radio buttons.  Therefore, please go ahead and mark this bug as complete, or Tested on UAT and we can then go ahead and close this.

**Comment by jill-iterators** [comment]
Now that space key is in order, this can be closed.
